homework11/task1.py

```diff
@@ -0,0 +1,59 @@
+"""
+Vasya implemented nonoptimal Enum classes.
+Remove duplications in variables declarations using metaclasses.
+from enum import Enum
+class ColorsEnum(Enum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+class SizesEnum(Enum):
+    XL = "XL"
+    L = "L"
+    M = "M"
+    S = "S"
+    XS = "XS"
+Should become:
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+assert ColorsEnum.RED == "RED"
+assert SizesEnum.XL == "XL"
+"""
+
+
+class SimplifiedEnum(type):
+    """
+        This metaclass creates class with attributes from __keys
+        and checks if they are valid.
+    """
+    def __new__(cls, *args, **kwargs):
```

Comment:
Just change these arguments to (cls, name, bases, dct) and you can get rid of "clsname, bases, attrs = args" line